dataprocspawner/spawner.py

```diff
@@ -603,33 +739,32 @@ def options_from_form(self, formdata):
 
       options[key] = value
 
-    self.log.info(f"""User selected cluster: {options.get('cluster_type')}
-          and zone: {self.zone} in region {self.region}.""")
+    self.log.info(
+        f'# User selected cluster: {options.get("cluster_type")} '
+        f'and zone: {self.zone} in region {self.region}.')
 
     return options
 
 ################################################################################
 # Overwrite
 ################################################################################
   def get_env(self):
-    """ Overwrites the original function to get a new Hub URL accessible by
-    Dataproc when JupyterHub runs on an AI Notebooks which by default would
-    return a local address otherwise.
+    """ Sets env variables for spawner server.
+
+    Overwrites the original function to overwrite some values and pass them to
+    the Dataproc cluster through dataproc:jupyter.hub.env. Most JupyterHub values
+    are now obsolete due to the redirect to prevent Dataproc cluster to be able
+    to communicate back to JupyterHub.
     """
     env = super().get_env()
-
-    # Sets in the jupyterhub_config related to ai notebook.
-    if 'NEW_JUPYTERHUB_API_URL' in env:
-      env['JUPYTERHUB_API_URL'] = env['NEW_JUPYTERHUB_API_URL']
-      env['JUPYTERHUB_ACTIVITY_URL'] = url_path_join(
-          env['NEW_JUPYTERHUB_API_URL'],
-          'users',
-          # tolerate mocks defining only user.name
-          getattr(self.user, 'escaped_name', self.user.name),
-          'activity',
-      )
-
-    self.log.info(f'env is {env}')
+    remove_env =  ['JUPYTERHUB_API_TOKEN', 'JPY_API_TOKEN', 'JUPYTERHUB_HOST',
```

Comment:
We need to keep at least the JUPYTERHUB_USER env variable here for the modified jupyterhub-singleuser server.

Reply:
Fixed in 7d23da939c1e101356da7a7df3871d20c47d2422